t/01_use.t

```diff
@@ -0,0 +1,9 @@
+use strict;
+use warnings;
+
+use Test::More;
+
+BEGIN { use_ok('Email::Stuffer::TestLinks'); }
```

Comment:
This file can be removed, the `Test::Compile` plugin handles this already

Reply:
Done